controllers/categories.js

```diff
@@ -14,4 +17,21 @@ module.exports = {
       next(error)
     }
   },
+  destroy: catchAsync(async (req, res, next) => {
+    try {
+      const deletedCategory = await deleteCategory(req.params.id)
+      endpointResponse({
+        res,
+        msg: 'Category was succesfully deleted',
+        body: deletedCategory,
+      })
+    } catch (error) {
+      next(
+        new ErrorObject(
+          `[Error deleting Category] - [category - delete]: ${error.message}`,
+          404,
```

Comment:
sure.. this 404 is the error code we want  ? maybe 500 ...

Reply:
You're right. Will check that out! Thanks!